src/wasm/literal.cpp

```diff
@@ -2039,6 +2023,26 @@ Literal Literal::eqI64x2(const Literal& other) const {
   return compare<2, &Literal::getLanesI64x2, &Literal::eq, int64_t>(*this,
                                                                     other);
 }
+Literal Literal::neI64x2(const Literal& other) const {
+  return compare<2, &Literal::getLanesI64x2, &Literal::ne, int64_t>(*this,
```

Comment:
Yes, this is OK. The default parameter for all other operations is `int32_t`, which is the type used to construct individual lanes even in i8x16 and i16x8 vectors (since we don't have i8 or i16 literals).